main.go

```diff
@@ -70,6 +147,37 @@ func getSwap() int {
 	return int(v)
 }
 
+func getCredit() int {
+
+	instanceId, err := getInstanceId()
+
+	if err != nil {
+		return 0
+	}
+
+	commandStr := "aws cloudwatch get-metric-statistics --namespace AWS/EC2 --metric-name CPUCreditBalance --region ap-southeast-2 --dimensions Name=InstanceId,Value=" + instanceId + " --start-time $(date -d '10 minute ago' +%s) --end-time=$(date +%s) --period 3600 --statistics Minimum --unit Count | jq '.Datapoints[0].Minimum'"
```

Comment:
I think start time should just be 1 minute ago? (ie this monitor runs once per minute)

Reply:
in the doc , it says `CPU credit metrics are available at a five-minute frequency only.`
I'll change to 5.